substrate/client/transaction-pool/src/graph/watcher.rs

```diff
@@ -113,6 +113,12 @@ impl<H: Clone, BH: Clone> Sender<H, BH> {
 	}
 
 	/// Transaction has been dropped from the pool because of the limit.
+	pub fn limit_enforced(&mut self) {
```

Comment:
dq: We plan in the future to introduce a new `TxStatus::LimitEnfoced`?

Reply:
Not really needed at the moment. We could. But, on the other hand less changes in API, probably better.
Don't have a strong opinion here.